cudapoa/src/batch.cpp

```diff
@@ -30,31 +30,47 @@ std::unique_ptr<Batch> create_batch(int32_t device_id,
                                     bool cuda_banded_alignment)
 {
     // a decision flag to determine proper type definition for ScoreT
-    const bool use_32_bit_for_ScoreT = use32bitInt(batch_size, gap_score, mismatch_score, match_score);
+    const bool use_32_bit_for_ScoreT = use32bitScore(batch_size, gap_score, mismatch_score, match_score);
 
     if (use_32_bit_for_ScoreT)
```

Comment:
any benefit to calling a variable instead of the function directly (like being done for `use32bitSize`?

Reply:
no, initially I had added it with a short comment, but the comment is moved to function declaration. changed.